### ISSUE TITLE
Issue 8 Hyperparameter Tuning

### ISSUE THREAD

**Issue by yfchen123** [yfchen123/Airline-Satisfaction-Analysis#7]
Perform hyperparameter tuning and benchmark the performance:
- We use grid or random search to perform the tuning (Note that gradient descent is more efficient but sub optimal you should ask the instructor if we can perform gradient descent to tune the parameters instead.).
- Compare the performance of the model before and after tuning.

**Comment by yfchen123** [comment]
The instructor says we should use known libraries, so do not do gradient descent on the parameter space as this is not a research project.

**Comment by yfchen123** [comment]
Marked as finished.
